test.md

```diff
@@ -270,6 +270,38 @@ print it and run it through the `markdownify` Liquid filter. Here's a demo:
 </div>
 </div>
 
+#### Synchronizing multiple tab groups
```

Comment:
Thanks for documenting this! I should have done it when I first created it 😅
Looking back at the script, I think we can can improve it so that you don't need to use the `data-group` attribute.

Do you think declaring tabs like I'm doing below is easier than what we have today?

```
<ul class="nav nav-tabs">
  <li class="active"><a data-toggle="tab" data-target="#go">Go</a></li>
  <li><a data-toggle="tab" data-target="#python">Python</a></li>
</ul>
...
<ul class="nav nav-tabs">
  <li class="active"><a data-toggle="tab" data-target="#go-1">Go</a></li>
  <li><a data-toggle="tab" data-target="#python-1">Python</a></li>
</ul>
...
```

Instead of using the `data-group` attribute, you would use `#go` in the first name, and then `#go-1`, `#go-2`...
This is a bit less explicit, so what do you think?